src/test/java/steps/LoginStep.java

```diff
@@ -0,0 +1,107 @@
+package steps;
+
+import base.BaseUtil;
+import pages.LoginPage;
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class LoginStep extends BaseUtil{
+
+    private BaseUtil base;
+
+    public LoginStep(BaseUtil base){
+        this.base = base;
+    }
+
+    @Given("^I navigate to the login page$")
+    public void iNavigateToTheLoginPage() throws Throwable {
+        base.driver.navigate().to("https://calendar.google.com/calendar");
+    }
+
+    @And("^I click on Next button$")
+    public void iClickOnNextButton() throws Throwable {
+        LoginPage page = new LoginPage(base.driver);
+        page.clickNext();
+    }
+
+    @Then("^I should see the password field$")
+    public void iShouldSeeThePasswordField() throws Throwable {
+        LoginPage page = new LoginPage(base.driver);
+        Thread.sleep(5000);
```

Comment:
You should use the WebDriverWait here instead of Thread.sleep(...)
new WebDriverWait(getWebDriver(), WAIT_PAGE_LOAD_SECONDS);